src/main/scala/com/microsoft/partnercatalyst/fortis/spark/ProjectFortis.scala

```diff
@@ -103,88 +86,90 @@ object ProjectFortis extends App {
   ssc.awaitTermination()
 
   /**
-    * TODO: Use Cassandra to populate
+    * TODO: Replace with cassandra configuration manager implementation
     * Build connector config registry from hard-coded values for demo.
     *
-    * The key is the name of the pipeline and the value is a list of connector configs whose streams should comprise it.
     */
-  private def getStreamRegistry: Map[String, List[ConnectorConfig]] = {
-    Map[String, List[ConnectorConfig]](
-      "instagram" -> List(
-        ConnectorConfig(
-          "InstagramTag",
-          Map(
-            "authToken" -> System.getenv("INSTAGRAM_AUTH_TOKEN"),
-            "tag" -> "rose"
+  private object DummyConfigurationManager extends ConfigurationManager {
+    override def fetchStreamConfiguration(pipeline: String): List[ConnectorConfig] = {
+      // The key is the name of the pipeline and the value is a list of connector configs whose streams should comprise it.
+      Map[String, List[ConnectorConfig]](
+        "instagram" -> List(
+          ConnectorConfig(
+            "InstagramTag",
+            Map(
+              "authToken" -> System.getenv("INSTAGRAM_AUTH_TOKEN"),
```

Comment:
Yes definitely. That will be a part of the broadcast and state update task that I should have a PR for this week.